weave/flow/eval.py

```diff
@@ -192,7 +202,9 @@ async def summarize(self, eval_table: EvaluationResults) -> dict:
                     summary[name] = model_output_summary
         return summary
 
-    async def get_eval_results(self, model: Union[Op, Model]) -> EvaluationResults:
+    async def get_eval_results(
+        self, model: Union[Op, Model], verbose: bool = False
```

Comment:
Should verbosity be controlled here or in settings?

Reply:
seems practical here